endpoints/_includes/functions.php

```diff
@@ -746,6 +748,30 @@ function helplineSearch($latitude, $longitude)
     return json_decode(get($search_url));
 }
 
+function getBMLTserver($latitude, $longitude, $search_state)
+{
+	$helpline_search_radius=setting('helpline_search_radius');
+	if ($search_state=="1") {
```

Comment:
Should use integers instead of strings.  More importantly, what does 1 and 2 mean?

Reply:
Switching to SearchType class